extensions/amp-pinterest/0.1/pin-widget.js

```diff
@@ -336,4 +342,17 @@ export class PinWidget {
 
     return this.fetchPin().then(this.renderPin.bind(this));
   }
+
+  /**
+   * Determine the height of the contents to allow resizing after first layout.
+   *
+   * @return {number|null}
+   */
+  height() {
+    return (
+      this.heightOwnerElement_
+        .getBoundingClientRect /* REVIEW */
```

Comment:
Does `/* REVIEW */` have any special meaning?

Reply:
`/* REVIEW */` is a pointer for the reviewer to ensure that the use of a restricted API is correct.

When submitting, this should be changed to `/* OK */`. Without `/* OK */` the PR won't be able to be merged.